src/AggregateFunctions/AggregateFunctionMinMaxAny.h

```diff
@@ -766,19 +766,24 @@ static_assert(
 
 
 /// For any other value types.
+template <bool IS_NULLABLE = false, bool IS_NULL_GREATER = false>
```

Comment:
Do we use `IS_NULL_GREATER` ? If i get right it's always `false`

Reply:
It's false at present.

Reply:
Let's remove it then?